test/TropicalGeometry/tropical_polyhedra.jl

```diff
@@ -0,0 +1,242 @@
+polymake_version = pkgversion(Polymake.polymake_jll)
+
+for minOrMax in (min,max)
+__sign = minOrMax == min ? 1 : -1
+oo = minOrMax == min ? inf : -inf
+
+@testset "TropicalPolyhedron{$minOrMax}" begin
```

Comment:
```suggestion
@testset "TropicalPolyhedron{$minOrMax}" for (minOrMax, __sign, oo) in [(min, 1, inf), (max, -1, -inf)]
```
some suggestion to make this more readable